Makefile

```diff
@@ -0,0 +1,45 @@
+build:
+	docker-compose build
+
+createuser: build
+	docker-compose run app python manage.py createsuperuser
+
+check_migrations: build
+	docker-compose run app python manage.py makemigrations --check --dry-run --noinput
+
+makemigrations: build
+	docker-compose run app python manage.py makemigrations
```

Comment:
nit: i just noticed these are tabs and we use spaces everywhere else.

Reply:
ignore this nit. make only accepts tabs. 🤮

Reply:
YYYYYyeeeeepppppp